oaknational/Oak-Web-Application
# ENG-492 fix: legacy canonical lesson + downloads

## Description

- Canonical lesson/download pages are currently 404ing. They aren’t currently linked to anywhere so it’s not urgent urgent, but should be fixed.

**Note:** when a canonical lesson has only one lesson pathway, the download links go the "in pathway" version of lesson downloads. If this is not desired, should ticket

## How to test

1. Go to https://deploy-preview-2011--oak-web-application.netlify.thenational.academy/teachers/lessons/introduction-to-epic-poetry-68u3gc
2. should not 404
3. Go https://deploy-preview-2011--oak-web-application.netlify.thenational.academy/teachers/lessons/introduction-to-epic-poetry-68u3gc/downloads
4. should not 404
5. should be no regressions of non-legacy content
